extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -549,7 +549,12 @@ export class AmpConsent extends AMP.BaseElement {
     user().assert(isJsonScriptTag(script),
         `${TAG} consent instance config should be put in a <script>` +
         'tag with type= "application/json"');
-    const config = parseJson(script.textContent);
+    let config;
+    try {
+      config = parseJson(script.textContent);
```

Comment:
Use `tryParseJson`: https://github.com/ampproject/amphtml/blob/90a3f16a680ec9177cf07f7a89616c60924737fc/extensions/amp-next-page/0.1/amp-next-page.js#L72-L74

Reply:
Switched to `tryParseJson`. But will need to call `user().assert()` because the function is supposed to throw at errors.